hardware_interface/include/hardware_interface/helpers.hpp

```diff
@@ -87,6 +119,25 @@ template <typename Container>
     [&container](const typename Container::key_type & item) { return has_item(container, item); });
 }
 
+/**
+ * @brief Check if the container has all of the items.
+ * @param container The container to search in.
+ * @param items The items to search for.
+ * @return True if the container has all of the items, false otherwise.
+ */
```

Comment:
isn't this the wrong docstring?

Reply:
you are right. Fixed :)

Reply:
wait, this is for `has_any_item`. You have changed L136 instead

Reply:
🙈 Right. Fixed :)